setup.py

```diff
@@ -18,6 +18,7 @@
         'navmazing==1.1.4',
         'pytest==3.4.0',
         'six==1.11.0',
+        'wait_for',
```

Comment:
that is probably already added in other PRs, so just make rebase for current one